### PR TITLE
add dbghelp for source, bt and print-string commands

### PR DESCRIPTION
**Pull request by therealdreg**
Add help for **three** existing commands: **bt**, **source** & **print-string**

helpadded

NOTE: **source** command is implemented in **dbg_main.cc** (for this reason I only add **BX_TOKEN_SOURCE** for **help source** command)

helpbt

### CODE REVIEW

File: bochs/bx_debug/parser.y [hunk 1]
```diff
@@ -1350,6 +1351,16 @@ help_command:
          dbg_printf("    Use it as a refresh/context.\n");
          free($1);free($2);
        }
+     | BX_TOKEN_HELP BX_TOKEN_PRINT_STRING '\n'
+       {
+         dbg_printf("print-string <laddr> - prints a null-ended string from a linear address.\n");
+         free($1);free($2);
+       }
```
Comment (stlintel) [inline bochs/bx_debug/parser.y:1358]: it looks like 'source' is a new command (just added token) but it is not implemented yet, the token is not used outside of 'help source'

File: bochs/bx_debug/dbg_main.cc [hunk 2]
```diff
@@ -3786,7 +3786,7 @@ void bx_dbg_print_help(void)
   dbg_printf("h|help command - show short command description\n");
   dbg_printf("-*- Debugger control -*-\n");
   dbg_printf("    help, q|quit|exit, set, instrument, show, trace, trace-reg,\n");
-  dbg_printf("    trace-mem, u|disasm, ldsym, slist, addlyt, remlyt, lyt\n");
+  dbg_printf("    trace-mem, u|disasm, ldsym, slist, addlyt, remlyt, lyt, source\n");
```
Comment (stlintel) [inline bochs/bx_debug/dbg_main.cc:3789]: please don't forget to update user docs too.
addlyt/remlyt is also missing there ...

**Review by stlintel — CHANGES_REQUESTED** [review]

File: bochs/bx_debug/parser.y (hunk 1 above)
Comment (therealdreg) [reply bochs/bx_debug/parser.y:1358]: @stlintel **source** its an old command **implemented in dbg_main.cc** (layout support use this feature) I am only adding documentation for this old command. I think its a very useful command for the user.

Comment (therealdreg) [reply bochs/bx_debug/parser.y:1358]: @stlintel  look this: 

oldcode

File: bochs/bx_debug/dbg_main.cc (hunk 2 above)
Comment (therealdreg) [reply bochs/bx_debug/dbg_main.cc:3789]: done!

**Comment by therealdreg** [comment]
user.dbk updated, bt help added, suggestions/fixes added :D

**Review by stlintel — APPROVED** [review]

**Pull request merged**